src/test/java/com/wikia/webdriver/testcases/adstests/TestUrlBuilder.java

```diff
@@ -10,271 +10,284 @@
+                    "de"
+            },
+            {
+                    "memory-alpha", "wiki/Portal:Main", false,
+                    "http://memory-alpha.wikia.com/wiki/Portal:Main",
+                    "http://memory-alpha.preview.wikia.com/wiki/Portal:Main",
+                    "http://memory-alpha.sandbox.wikia.com/wiki/Portal:Main",
+                    "http://memory-alpha.sandbox-mercurydev.wikia.com/wiki/Portal:Main",
+                    "http://memory-alpha.dmytror.wikia-dev.pl/wiki/Portal:Main",
+                    "http://memory-alpha.wikia-staging.com/wiki/Portal:Main",
+                    "en"
+            },
+            {
+                    "memory-alpha", "wiki/Hauptseite", false,
+                    "http://memory-alpha.wikia.com/de/wiki/Hauptseite",
+                    "http://memory-alpha.preview.wikia.com/de/wiki/Hauptseite",
+                    "http://memory-alpha.sandbox.wikia.com/de/wiki/Hauptseite",
+                    "http://memory-alpha.sandbox-mercurydev.wikia.com/de/wiki/Hauptseite",
+                    "http://memory-alpha.dmytror.wikia-dev.pl/de/wiki/Hauptseite",
+                    "http://memory-alpha.wikia-staging.com/de/wiki/Hauptseite",
+                    "de"
+            },
+    };
 
-    for (Object[] data : TEST_DATA) {
-      Assertion.assertEquals(new UrlBuilder("prod", (Boolean) data[2], (Boolean) data[3])
-                                 .getUrlForPath((String) data[0], (String) data[1]),
-                             (String) data[4]);
-      Assertion.assertEquals(new UrlBuilder("preview", (Boolean) data[2], (Boolean) data[3])
-                                 .getUrlForPath((String) data[0], (String) data[1]),
-                             (String) data[5]);
-      Assertion.assertEquals(new UrlBuilder("sandbox", (Boolean) data[2], (Boolean) data[3])
-                                 .getUrlForPath((String) data[0], (String) data[1]),
-                             (String) data[6]);
-      Assertion.assertEquals(new UrlBuilder("sandbox-mercurydev", (Boolean) data[2], (Boolean) data[3])
-                                 .getUrlForPath((String) data[0], (String) data[1]),
-                             (String) data[7]);
-      Assertion.assertEquals(new UrlBuilder("dev-dmytror", (Boolean) data[2], (Boolean) data[3])
-                                 .getUrlForPath((String) data[0], (String) data[1]),
-                             (String) data[8]);
-      Assertion.assertEquals(new UrlBuilder("staging", (Boolean) data[2], (Boolean) data[3])
-                                 .getUrlForPath((String) data[0], (String) data[1]),
-                             (String) data[9]);
+    @Test(groups = {"TestUrlBuilder", "unitTests"})
+    public void urlBuilderTests() {
+        Configuration.setTestValue("qs", "");
+
+        for (Object[] data : TEST_DATA) {
+            System.out.println("Nazwa wikii: " + data[0]);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS106)